app/src/test/java/com/hover/stax/utils/UtilsTest.kt

```diff
@@ -1,10 +1,10 @@
 package com.hover.stax.utils
 
-import org.junit.Assert
+import org.junit.Assert.*
```

Comment:
Thanks for making this better; instead of a wildcard, can we use single imports?

Reply:
I wanted to actually enforce this with detekt

But fixing it here (my formatting defaults to wildcards)